### PR TITLE
chore(rustfmt): remove executable path from usage string

### LINKED ISSUE #5214: rustfmt binary path in usage string
`rustfmt` help command returns usage information in which `rustfmt` binary path is appended to the usage message. The usage message will be cleaner without the binary's path. I am happy to raise a pr for this if we are okay to remove the path from the usage string.
Screenshot 2022-02-03 at 9 22 29 PM

### PR DESCRIPTION
**Pull request by tharun208**
This PR removes the `rustfmt` executable path from usage string.

Fixes #5214

### CODE REVIEW

**Comment by ytmimi** [comment]
Thanks for another PR!

The proposed changes look good to me.

One minor ask: If this is the desired usage text for rustfmt I think it would be nice to have a test for it. Would you mind including a test for the usage text in ``tests\rustfmt\main.rs``. You should be able to use the existing tests as a guide.

**Comment by tharun208** [comment]
@ytmimi added test for usage text

File: tests/rustfmt/main.rs [hunk 1]
```diff
@@ -106,3 +106,13 @@ fn inline_config() {
...
 }
+
+#[test]
+fn rustfmt_help() {
+    let args = [
+        "--help",
+    ];
+    let (stdout, _) = rustfmt(&args);
+    assert!(stdout.contains(&format!("Format Rust code")));
+    assert!(stdout.contains(&format!("usage: rustfmt [options] <file>...")));
```
Comment (ytmimi) [inline tests/rustfmt/main.rs:117]: To show that we're not including the path of the rustfmt executable would it not be easier to do a single check for the string ``"Format Rust code\n\nusage: rustfmt [options] <file>..."``?

File: tests/rustfmt/main.rs [hunk 2]
```diff
@@ -106,3 +106,13 @@ fn inline_config() {
             && contains("format_strings = true")
     );
 }
+
+#[test]
+fn rustfmt_help() {
```
Comment (ytmimi) [inline tests/rustfmt/main.rs:111]: a little nit picky, but this isn't really a full test of the help text, its more specifically a test of the usage text. Would it be possible to rename this to be more specific?

Comment (tharun208) [reply tests/rustfmt/main.rs:111]: sure will rename it

**Comment by ytmimi** [comment]
Thanks again for this PR!

The changes look good to me, and I'm happy to move forward with them!

**Review by ytmimi — APPROVED** [review]

**Review by calebcartwright — APPROVED** [review]
LGTM too! Thanks for the PR @tharun208 and @ytmimi for reviewing!

**Pull request merged**